homework-submission/week2/src/App.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import './App.css';
+import uuid from 'uuid';
+import List from './components/list';
+import ListHeader from './components/listHeader';
+import Input from './components/Input';
+import Button from './components/Button';
+
+function App() {
+  const [Todos, setTodos] = useState([
```

Comment:
In Javascript we don't write variable name with a capital letter (`Todos`), only classes are written with a capital